Fable.Remoting.AwsLambda/FableLambdaAdapter.fs

```diff
@@ -0,0 +1,173 @@
+﻿namespace Fable.Remoting.AwsLambda.Worker
+
+open System
+open System.Net
+open System.Text
+open System.Threading.Tasks
+open System.IO
+open Fable.Remoting.Server
+open Fable.Remoting.Server.Proxy
+open Amazon.Lambda.APIGatewayEvents
+open Newtonsoft.Json
+
+type HttpRequestData = APIGatewayHttpApiV2ProxyRequest
+type HttpResponseData = APIGatewayHttpApiV2ProxyResponse
```

Comment:
I've aliased the AWS types for parity with the Azure Functions implementation to make comparison easier, but they don't need to remain.